storage_service/locations/models/package.py

```diff
@@ -189,6 +189,7 @@ def __init__(self, *args, **kwargs):
         self.local_path = None
         self.local_path_location = None
         self.origin_location = None
+        self.local_tempdirs = []
```

Comment:
This could be a set if you wanted to ensure unique values? In practice I don't think it'll make a difference.